docs/pages/eas/workflows/syntax.mdx

```diff
@@ -361,34 +545,37 @@ custom_job:
 Runs the `expo prebuild` command using the package manager (bun, npm, pnpm, or Yarn) detected based on your project with the command best suited for your build type and build environment.
 
 ```yaml
-custom_job:
-  steps:
-    - uses: eas/checkout
-    - uses: eas/install_node_modules
-    # @info #
-    - uses: eas/prebuild
-    # @end #
+jobs:
+  my_job:
+    steps:
+      - uses: eas/checkout
+      - uses: eas/install_node_modules
+      # @info #
+      - uses: eas/prebuild
+      # @end #
 ```
 
 ```yaml
-custom_job:
-  steps:
-    - uses: eas/checkout
-    - uses: eas/install_node_modules
-    - uses: eas/resolve_apple_team_id_from_credentials:
+jobs:
+  my_job:
+    steps:
+      - uses: eas/checkout
+      - uses: eas/install_node_modules
+      - uses: eas/resolve_apple_team_id_from_credentials
         id: resolve_apple_team_id_from_credentials
-    # @info #
-    - uses: eas/prebuild
-      with:
-        clean: false
-        apple_team_id: ${{ steps.resolve_apple_team_id_from_credentials.outputs.apple_team_id }}
-    # @end #
+      # @info #
+      - uses: eas/prebuild
+        inputs:
+          clean: false
+          apple_team_id: ${{ steps.resolve_apple_team_id_from_credentials.outputs.apple_team_id }}
+      # @end #
 ```
 
-| Input           | Type      | Description                                                                                                                              |
-| --------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| `clean`         | `boolean` | Optional input defining whether the function should use `--clean` flag when running the command. Defaults to false.                      |
-| `apple_team_id` | `boolean` | Optional input defining Apple team ID which should be used when doing prebuild. It should be specified for iOS builds using credentials. |
+| Property               | Type      | Description                                                                                                                              |
```

Comment:
I think we can leave the `main` version of this table (I amended it today).

- `name` is not a `prebuild`-specific property so there's little benefit in mentioning it here
- we don't mention `inputs` which is a Custom Builds concept
- the table looks more like the table for `send_slack_message` where we only list "Input"